lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -129,6 +129,14 @@ if Enum.all?(
                   Tells if the session is live or a vod type of broadcast. Use live when you generate HLS stream from real-time data.
                   Use vod when generating HLS from file.
                   """
+                ],
+                video_layout_module: [
+                  spec: module(),
+                  default: Membrane.RTC.Engine.Endpoint.HLS.MobileLayoutMaker,
+                  description: """
+                  Module implementing `Membrane.RTC.Engine.Endpoint.HLS.VideoLayoutMaker` behavior
+                  that should be used by the HLS endpoint.
+                  """
```

Comment:
I think that this should be a field in `compositor_config` because passing it with `compositor_config` equal to nil is useless.